query-service/src/main/resources/configs/common/application.conf

```diff
@@ -18,6 +17,7 @@ service.config = {
       type = pinot
       clientConfig = zookeeper
       requestHandlerInfo = {
+        tenantColumnName = tenant_id
```

Comment:
Why did you move this into the request handler info? Seems wrong to me since it opens up more config bug opportunities.

Reply:
Because this is the right design. 
Imagine there are different data stores are present and there are handlers for each store, table combo. Do you want to dictate the same column name across all tables and stores? The tenant_id column belongs at the handler level so that they can choose whatever they want to call it as.

Reply:
I'd understand there were multiple data sources but we are not there yet. Also if you wanna go this route you will need to make changes to helm-values repo.

Reply:
Replied to you separately for this.